### ISSUE TITLE
about paper or support ducuments

### ISSUE THREAD

**Issue by lxy-94** [niekverw/Deep-Learning-Based-ECG-Annotator#2]
Hi, 
I'm very interesting to this project and hope to understand this project further. Have you written some papers or support documents explaining this project? .

**Comment by niekverw** [comment]
No,  I haven't found the time for this, but a proper paper and comparative analysis is on my todo list. Any help is appreciated
